.github/workflows/development-environment.yml

```diff
@@ -89,10 +89,9 @@ jobs:
       - name: Checkout sentry
         uses: actions/checkout@v2
 
-      # We update brew because we need a newer version of pyenv which has access to 3.8.13
       - name: Install pyenv
         run: |
-          brew update && brew install pyenv
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install pyenv
```

Comment:
At the time the original line got added the version of pyenv we needed required an update.

If the checks fail, you may want to add `|| brew update && brew install pyenv`.
If it all works, go ahead and merge.

Thanks Anthony!